certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+            "$ref": "#/definitions/AIForged.DAL.Models.ParameterDefSettingViewModel"
+          }
+        },
+        "totalCharge": {
+          "type": "number",
+          "format": "double"
+        },
+        "userCount": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "parameterCount": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "documentCount": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "validation": {
+          "type": "string"
+        }
+      }
+    },
+    "AIForged.DAL.ParameterDefinitionStatus": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "Default",
+        "Deleted"
+      ],
+      "enum": [
+        0,
+        99
+      ]
+    },
+    "AIForged.DAL.ParameterDefinitionCategory": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "None",
+        "Setting",
+        "Service",
+        "Statistics",
+        "Results",
+        "Dependency",
+        "Verification",
+        "DataSet",
+        "PreProcessor",
+        "Processor",
+        "PostProcessor",
+        "Extraction"
+      ],
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        20,
+        21,
+        22,
+        40
+      ]
+    },
+    "AIForged.DAL.GroupingType": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "None",
+        "Page",
+        "Cluster",
+        "Table",
+        "Row",
+        "Column",
+        "Paragraph",
+        "Line",
+        "Word",
+        "Array",
+        "Form",
+        "Field",
+        "Anchor",
+        "Unknown"
+      ],
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        7,
+        8,
+        9,
+        10,
+        11,
+        12,
+        13,
+        99
+      ]
+    },
+    "AIForged.DAL.ValueType": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "None",
+        "String",
+        "Number",
+        "Currency",
+        "Boolean",
+        "DateTime",
+        "TimeSpan",
+        "Email",
+        "Enum",
+        "Picture",
+        "Password",
+        "SecureString",
+        "Address",
+        "IDNo",
+        "Country",
+        "Code",
+        "TelephoneNo",
+        "GPS",
+        "ServiceType",
+        "Service",
+        "Data",
+        "DataSet",
+        "System",
+        "Unknown"
+      ],
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        11,
+        12,
+        13,
+        14,
+        15,
+        17,
+        18,
+        19,
+        20,
+        90,
+        91,
+        98,
+        99
+      ]
+    },
+    "AIForged.DAL.RequiredOption": {
+      "type": "integer",
+      "description": "",
```

Comment:
@aiforged Empty property values can be removed until unless you have a specific reason to keep them.